.github/workflows/azure-static-web-apps-black-water-0eaf5100f.yml

```diff
@@ -44,6 +44,21 @@ jobs:
           container_name: $web
           source_folder: /home/runner/work/app-store-badge/app-store-badge/dist/ms-store-badge.bundled.js
 
+  deploy_images:
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+    needs: build_and_deploy_job
+    name: Upload Images to Blob Storage
+    steps:
+      - uses: actions/checkout@v2
+      - uses: LanceMcCarthy/Action-AzureBlobUpload@v1.9.0
+        with:
+          connection_string: ${{ secrets.AZURE_BLOB_STORAGE_CONNECTION_STRING }}
+          container_name: $web
+          source_folder: /home/runner/work/app-store-badge/app-store-badge/src/images

```

Comment:
Two things here.

1. Instead of copying from /src/images, can you copy from /dist/images? That way, everything that gets deployed comes from /dist.
2. I think images aren't being copied to /dist on build. Can you update this in our rollup file so that images get copied to /dist?